pageserver/src/tenant/mgr.rs

```diff
@@ -206,20 +256,48 @@ pub async fn init_tenant_mgr(
                         }
                     };
 
+                    // Try loading the location configuration
+                    let mut location_conf = match Tenant::load_tenant_config(conf, &tenant_id) {
+                        Ok(c) => c,
+                        Err(e) => {
+                            warn!("Error loading tenant config, skipping the tenant ({e})");
+                            continue;
```

Comment:
Hm, if we skip the tenant here, it will be 404 from console's perspective.
It doesn't expect that we lose tenants.
I think in the current state of the codebase, we should make the tenant Broken.
But we need a tenant conf to construct a Tenant object.
Arghh.

Also, if we skip here, we still have the on-disk state.
Subsequent attach attempts will fail.
Not good.

I guess we should do the following:
* if it's a filesystem error: crash the pageserver
* if the tenant conf is corrupt (deserialize error): nuke the tenant dir and continue ?

Reply:
Fortunately we can put a tenant in a broken state from here, one doesn't need  a LocationConf.  9fcea2dbd